src/main/java/uk/gov/hmcts/reform/sendletter/config/FlywayConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.sendletter.config;
+
+import org.flywaydb.core.Flyway;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.flyway.FlywayMigrationStrategy;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.sendletter.database.migration.FlywayNoOpStrategy;
+
+import javax.sql.DataSource;
+
+@Configuration
+@ConditionalOnClass(Flyway.class)
+@ConditionalOnBean(DataSource.class)
+@ConditionalOnProperty(prefix = "flyway", name = "enabled", matchIfMissing = true)
```

Comment:
so with `flyway.enabled: true` Spring will run real migrations?

Reply:
spring will not configure flyway at all. this is a take out from spring's `FlywayAutoConfiguration`